<?xml version="1.0"?>
<launch>
    <!-- this is comment-->

    <node name="Client" pkg="multimodal_pedestrian_detection" type="client_multimodal" output="screen">
	<remap from="image_in" to="snr/scam/wide/left/image_color"/> 
<!--         <remap from="image_in" to="Image_In"/> -->
        <param name="classifier" value="/home/rui/workingcopies/lar4/src/perception/pedestrians/multimodal_pedestrian_detect/trained_boost_15Kf_2000w_19Ks_m8_M64_boot3.xml"/>
        <param name="classifier_result" value="/home/rui/Documents/2013_PedroSilva/PedestrianDetect/Results/RuiExamples/Frame"/>
    </node>

<!--    <node name="image_view" pkg="image_view" type="image_view"> 
        <remap from="image" to="xb3_short/left/image_raw"/> 
         <remap from="image" to="Image_Out"/>  
    </node> 
    -->
</launch>